rerun_py/rerun_sdk/rerun/log_deprecated/transform.py

```diff
@@ -103,30 +117,34 @@ def log_view_coordinates(
 
     """
     recording = RecordingStream.to_native(recording)
+    from rerun.archetypes import ViewCoordinates
+    from rerun.experimental import log
+
     if xyz == "" and up == "":
         _send_warning("You must set either 'xyz' or 'up'. Ignoring log.", 1)
         return
     if xyz != "" and up != "":
         _send_warning("You must set either 'xyz' or 'up', but not both. Dropping up.", 1)
         up = ""
     if xyz != "":
-        bindings.log_view_coordinates_xyz(
-            entity_path,
-            xyz,
-            right_handed,
-            timeless,
-            recording=recording,
-        )
+        xyz = xyz.upper()
+        if hasattr(ViewCoordinates, xyz):
+            log(entity_path, getattr(ViewCoordinates, xyz), timeless=timeless, recording=recording)
+        else:
+            raise ValueError("Invalid xyz value.")
     else:
         if right_handed is None:
             right_handed = True
-        bindings.log_view_coordinates_up_handedness(
-            entity_path,
-            up,
-            right_handed,
-            timeless,
-            recording=recording,
-        )
+
+        if (up, right_handed) not in _up_attrs:
+            raise ValueError("Invalid up value.")
+
+        up = _up_attrs[(up, right_handed)]
```

Comment:
should we validate the `up` argument first for a more clean error message if it's an invalid string?